### PR TITLE
[Dlight] Fix NormalizePrimFunc with scalar block

### PR DESCRIPTION
**Pull request by vincentccc**
fix NormalizePrimFunc when their is a standalone scalar block
```
@T.prim_func
def main(p0: T.Buffer((), "int32"), T_stack: T.Buffer((T.int64(3),), "int32")):
    T.func_attr({"tir.noalias": T.bool(True)})
    # with T.block("root"):
    compile_engine_const = T.alloc_buffer((), "int32")
    compile_engine_const_1 = T.alloc_buffer((), "int32")
    with T.block("compile_engine_const"):
        vi = T.axis.spatial(1, T.int64(0))
        T.reads()
        T.writes(compile_engine_const[()])
        compile_engine_const[()] = 16
    with T.block("compile_engine_const_1"):
        vi = T.axis.spatial(1, T.int64(0))
        T.reads()
        T.writes(compile_engine_const_1[()])
        compile_engine_const_1[()] = 20
    for ax0 in range(T.int64(3)):
        with T.block("T_stack"):
            v_ax0 = T.axis.spatial(T.int64(3), ax0)
            T.reads(compile_engine_const[()], p0[()], compile_engine_const_1[()])
            T.writes(T_stack[v_ax0])
            T_stack[v_ax0] = T.if_then_else(v_ax0 == T.int64(2), compile_engine_const[()], T.if_then_else(v_ax0 == T.int64(1), p0[()], compile_engine_const_1[()]))
```

### CODE REVIEW

**Comment by Hzfengsy** [comment]
cc @junrushao @vinx13

File: tests/python/dlight/test_primitives.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# pylint: disable=missing-docstring
+
+import tvm
+import tvm.testing
+from tvm import relay
+
+
+@tvm.testing.requires_cuda
+def test_normalize_primfunc_with_scalar():
+    relay_text = """
```
Comment (tqchen) [inline tests/python/dlight/test_primitives.py:26]: let us create a test case directly with tvmscript

**Review by tqchen — CHANGES_REQUESTED** [review]

**Comment by tqchen** [comment]
thank you @vincentccc can you create a testcase directly with tvmscript without going through relay

**Comment by vincentccc** [comment]
> thank you @vincentccc can you create a testcase directly with tvmscript without going through relay. It would also be great to simplify this case a bit further

Thanks for your time!
Sure, I will fix it

**Pull request merged**